src/frontends/paddle/src/op/dequantize_linear.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs dequantize_linear(const NodeContext& node) {
+    // extract the INPUTS
+    const auto x = node.get_input("X");
+    const auto scale = node.get_input("Scale");  // type: float or 1-D
+    const auto zero_point = node.get_input("ZeroPoint");
+
+    // assert shape of scale and zero_point
+    PADDLE_OP_CHECK(node, scale.get_partial_shape().rank().is_static(), "dequantize: scale rank must be static!");
+    PADDLE_OP_CHECK(node,
+                    zero_point.get_partial_shape().rank().is_static(),
+                    "dequantize: zero_point rank must be static!");
+    const auto& scale_shape = scale.get_partial_shape();
+    const auto& scale_shape_length = scale_shape.rank().get_length();
+
+    if (scale_shape_length == 1) {
```

Comment:
Input "Scale" could be float or 1-d tensor. Then the rank would be 0 or 1. Would you please explain a bit why do you check scale_shape_length = 1 and scale_shape_length =2 instead?

Reply:
> Input "Scale" could be float or 1-d tensor. Then the rank would be 0 or 1. Would you please explain a bit why do you check scale_shape_length = 1 and scale_shape_length =2 instead?

I check it. It seems a mistake in PaddleSlim docs. In the docs, the scale is described as float or 1-D while it is 1-D or 2-D in the real model.